src/website/src/app/documentation/get-started/get-started.component.ts

```diff
@@ -30,7 +30,8 @@ export class AppModule { }
 const HTML_IMPORTS = `
 <link rel="stylesheet" href="path/to/node_modules/@clr/ui/clr-ui.min.css">
 <link rel="stylesheet" href="path/to/node_modules/@clr/icons/clr-icons.min.css">
-<script src="path/to/node_modules/@webcomponents/custom-elements/custom-elements.min.js"></script>
+<script src="path/to/node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
```

Comment:
I wasn't aware of these dependency changes. Do people need both of these dependencies in all cases?

Reply:
This is the recommendation from @coryrylan that will be used with Core, so yes I think so.

Reply:
Also, the schematic will fix this for anyone automatically, and updated the ng-add schematic too.